README.md

```diff
@@ -1,2 +1,54 @@
-# hear-baseline
-Simple baseline model for the HEAR 2021 NeurIPS competition
+![HEAR2021](https://neuralaudio.ai/assets/img/hear-header-sponsor.jpg)
+# HEAR 2021 Baseline
+
+A simple DSP-based audio embedding consisting of a Mel-frequency spectrogram followed
+by a random projection. Serves as the baseline model for the HEAR 2021 and implements
+the [common API](https://neuralaudio.ai/hear2021-holistic-evaluation-of-audio-representations.html#common-api)
+required by the competition evaluation.
```

Comment:
Maybe add that a RP is less efficient than a CNN but is one of the simplest possible models code-wise.

Reply:
Also add a quick note how the weight file was generated

Reply:
Sounds good